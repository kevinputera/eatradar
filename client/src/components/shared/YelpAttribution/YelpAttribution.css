.yelp-attribution {
  position: relative;
  top: -2px;
  left: -10px;
  height: 40px;
}
