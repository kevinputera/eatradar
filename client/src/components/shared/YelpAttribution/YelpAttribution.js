import React from 'react';

import './YelpAttribution.css';
import attributionImg from './Yelp_trademark_RGB_outline.png';

function YelpAttribution(props) {
  return (
    <a
      href="https://www.yelp.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        src={attributionImg}
        alt="Yelp trademark"
        className="yelp-attribution"
      />
    </a>
  );
}

export default YelpAttribution;
