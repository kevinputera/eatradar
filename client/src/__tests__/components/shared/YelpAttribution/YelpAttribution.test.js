import React from 'react';
import { create } from 'react-test-renderer';

import YelpAttribution from '../../../../components/shared/YelpAttribution/YelpAttribution';

describe('tests for YelpAttribution component', () => {
  test('render must match snapshot', () => {
    const tree = create(<YelpAttribution />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
