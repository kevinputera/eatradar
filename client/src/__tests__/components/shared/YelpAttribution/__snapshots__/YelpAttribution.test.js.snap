// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for YelpAttribution component render must match snapshot 1`] = `
<a
  href="https://www.yelp.com/"
  rel="noopener noreferrer"
  target="_blank"
>
  <img
    alt="Yelp trademark"
    className="yelp-attribution"
    src="Yelp_trademark_RGB_outline.png"
  />
</a>
`;
